paddle/fluid/framework/op_info.h

```diff
@@ -65,8 +65,14 @@ struct OpInfo {
   }
 
   const GradOpMakerFN& GradOpMaker() const {
-    PADDLE_ENFORCE_NOT_NULL(grad_op_maker_,
-                            "Operator GradOpMaker has not been registered.");
+    std::string type = proto_ ? proto_->type() : "unknown";
```

Comment:
It is a bit confusing, why the proto_ maybe nullptr here?

Reply:
I don't know, just add it here for safety...